content/middlewares.md

```diff
@@ -132,6 +132,8 @@ forRoutes({ path: 'ab*cd', method: RequestMethod.ALL });
 
 The `'ab*cd'` route path will match `abcd`, `ab_cd`, `abecd`, and so on. The characters `?`, `+`, `*`, and `()` may be used in a route path, and are subsets of their regular expression counterparts. The hyphen ( `-`) and the dot (`.`) are interpreted literally by string-based paths.
 
+> info **Hint** Route wildcards do not work with fastify platform applications, since fastify [does not support wildcard routes](https://github.com/nestjs/nest/issues/972#issuecomment-516707805).
```

Comment:
I feel this should be `> warning **Warning**`

Reply:
Agree